sos/plugins/memory.py

```diff
@@ -37,7 +37,7 @@ def setup(self):
         self.add_cmd_output("free", root_symlink="free")
         self.add_cmd_output([
             "free -m",
-            "swapon --bytes --show"
+            "swapon --summary --verbose"
```

Comment:
Yeah, I had a feeling it would be a Debian thing... :-)

It's a shame really - there are a _lot_ of very useful things in more up-to-date packages and it's really only Debian (and Ubuntu due to the historical relationship) that tend to be the "hold outs".

Anyway, if you're happy to carry a patch for this, that's great, although if you did want to make a change in the plugin for Debian and Ubuntu users as I said, I wouldn't have any objection to that.